ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -236,7 +244,7 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
 
   return <div className="d-flex bg-white pb-5">
     <div className="d-flex flex-column flex-grow-1 mx-1 mb-1">
-      <div className="d-flex p-2 align-items-center">
```

Comment:
style nit: removing `align-items-center` caused the button text to no longer be centered